src/main/kotlin/persistence/repo/Module.kt

```diff
@@ -1,24 +1,24 @@
-package persistence
+package persistence.repo
 
+import dagger.Module
+import dagger.Provides
 import data.Language
 import data.User
 import data.dao.Dao
-import data.persistence.TrDatabase
 import io.requery.Persistable
 import io.requery.sql.KotlinConfiguration
 import io.requery.sql.KotlinEntityDataStore
 import io.requery.sql.SchemaModifier
 import io.requery.sql.TableCreationMode
 import org.sqlite.SQLiteDataSource
 import persistence.model.Models
-import persistence.repo.LanguageRepo
-import persistence.repo.UserRepo
+import javax.inject.Singleton
 
-class TrDatabaseImpl: TrDatabase {
-    private val dataStore: KotlinEntityDataStore<Persistable>
-    private val userRepo: Dao<User>
-    private val languageRepo: Dao<Language>
-    init {
+@Module
+class Module {
```

Comment:
Module is too ambiguous of a class name for this

Reply:
Got it. That was changed to DaoModule

Reply:
How about PersistenceModule?